src/fromager/wheels.py

```diff
@@ -129,6 +131,73 @@ def analyze_wheel_elfdeps(
     return requires, provides
 
 
+def add_extra_metadata_to_wheels(
+    ctx: context.WorkContext,
+    req: Requirement,
+    version: Version,
+    extra_environ: dict[str, str],
+    sdist_root_dir: pathlib.Path,
+    wheel_file: pathlib.Path,
+) -> None:
+    extra_data_plugin = overrides.find_override_method(
```

Comment:
Why not find_and_invoke()?

Reply:
We could have a default function that returns an empty dict, but maybe it's not worth it.